cms/jinja2/templates/base.html

```diff
@@ -38,73 +38,6 @@
 {% set navigation_settings = settings.navigation.NavigationSettings %}
 {% set main_menu = main_menu | default(navigation_settings.main_menu) %}
 
-{% set keyLinksList = [] %}
-
-{% if main_menu and main_menu.highlights %}
```

Comment:
Query, do you still need to check for the existence of the main_menu and highlights before passing the config through?

Reply:
that is covered https://github.com/ONSdigital/dis-wagtail/pull/85/files#diff-bfd42dbff73bbdea18630c438176ac08db2c376e9a8c3205a430e5f15caa3264R75

Reply:
Nope Helen, due to us doing the validation in Python. Which Dan kindly pointed to =)